<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuantifyMeasurement_Dlg</class>
 <widget class="QDialog" name="QuantifyMeasurement_Dlg">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/resources.qrc">
    <normaloff>:/new/prefix2/Ico/ren.ico</normaloff>:/new/prefix2/Ico/ren.ico</iconset>
  </property>
  <widget class="QPushButton" name="Btn_IsZero">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>100</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>校零</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>401</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>标准曲线</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_Save">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>280</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_MakeLine">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>220</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>生成曲线</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_Measure">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>160</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>测量</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_Quit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>340</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_Measureself">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>401</width>
     <height>241</height>
    </rect>
   </property>
   <property name="title">
    <string>测量区</string>
   </property>
   <widget class="QLineEdit" name="lineEdit_GoalCon_1">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>浓度输入1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Abs1">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>60</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_51">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>60</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>测量结果1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_52">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>90</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>测量结果2</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Abs2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>90</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>浓度输入2</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_GoalCon_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>90</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_53">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>120</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>测量结果3</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Abs3">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>120</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>浓度输入3</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_GoalCon_3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>120</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_54">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>150</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>测量结果4</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Abs4">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>150</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>浓度输入4</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_GoalCon_4">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>150</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_55">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>180</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>测量结果5</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Abs5">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>180</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>浓度输入5</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_GoalCon_5">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>180</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_56">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>210</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>测量结果6</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Abs6">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>210</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>浓度输入6</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_GoalCon_6">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>210</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_GoalconInput">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>完成浓度输入</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_Sum_Goalcon">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>目标浓度</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>10</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>计算公式</string>
   </property>
   <widget class="QComboBox" name="comboBox_FormulaMode">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Y=BX+A</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Y=CX2+BX+A</string>
     </property>
    </item>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>comboBox_Sum_Goalcon</tabstop>
  <tabstop>lineEdit_GoalCon_1</tabstop>
  <tabstop>lineEdit_Abs1</tabstop>
  <tabstop>lineEdit_GoalCon_2</tabstop>
  <tabstop>lineEdit_Abs2</tabstop>
  <tabstop>lineEdit_GoalCon_3</tabstop>
  <tabstop>lineEdit_Abs3</tabstop>
  <tabstop>lineEdit_GoalCon_4</tabstop>
  <tabstop>lineEdit_Abs4</tabstop>
  <tabstop>lineEdit_GoalCon_5</tabstop>
  <tabstop>lineEdit_Abs5</tabstop>
  <tabstop>lineEdit_GoalCon_6</tabstop>
  <tabstop>lineEdit_Abs6</tabstop>
  <tabstop>Btn_GoalconInput</tabstop>
  <tabstop>comboBox_FormulaMode</tabstop>
  <tabstop>Btn_IsZero</tabstop>
  <tabstop>Btn_Measure</tabstop>
  <tabstop>Btn_MakeLine</tabstop>
  <tabstop>Btn_Save</tabstop>
  <tabstop>Btn_Quit</tabstop>
 </tabstops>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
